guides/doc-Upgrading_and_Updating/topics/upgrading_a_disconnected_satellite.adoc

```diff
@@ -5,10 +5,29 @@
 Use this procedure for a {ProjectServer} not connected to the Red{nbsp}Hat Content Delivery Network.
 
 [WARNING]
-If you customize configuration files, manually or using a tool such as Hiera, these changes are overwritten when you enter the `{foreman-maintain}` command during upgrading or updating.
+====
+* If you customize configuration files, manually or using a tool such as Hiera, these changes are overwritten when you enter the `{foreman-maintain}` command during upgrading or updating.
 You can use the `--noop` option with the `{foreman-installer}` command to review the changes that are applied during upgrading or updating.
 For more information, see the Red Hat Knowledgebase solution https://access.redhat.com/solutions/3351311[How to use the noop option to check for changes in {Project} config files during an upgrade].
 
+* The hammer import and export commands have been replaced with `hammer content-import` and `hammer content-export` tooling.
++
+If you have scripts that are using: 
++
+----
+hammer content-view version export
+hammer content-view version export-legacy
+hammer repository export
+----
```

Comment:
Please have a look at another example file, e.g https://raw.githubusercontent.com/theforeman/foreman-documentation/master/guides/common/modules/proc_promoting-a-content-view-across-all-life-cycle-environments-within-an-organization.adoc

Two things you should change: add a `# ` prefix to the hammer commands; and specify the code block. In your case, a `[options="nowrap"]` is enough because you're not using any attribute substitution.

Reply:
This is resolved no, we do not use code syntax highlighting much but it is client side only for HTML and ignored for PDF.